fee_allocator/accounting/decorators.py

```diff
@@ -0,0 +1,56 @@
+from functools import wraps
```

Comment:
See comments below. I don't think these decorators or this style of handling rouding makes sense.